<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* background: url(/home/matt/Documents/imgpy/mantidimaging/gui/ui/code.jpg)*/</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWorkflow">
    <property name="title">
     <string>Workflow</string>
    </property>
    <addaction name="actionFilters"/>
    <addaction name="actionSavuFilters"/>
    <addaction name="actionCorTilt"/>
    <addaction name="actionTomopyRecon"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOnlineDocumentation"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWorkflow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionTomopy">
   <property name="text">
    <string>Tomopy</string>
   </property>
  </action>
  <action name="actionCorTilt">
   <property name="text">
    <string>Find COR and Tilt</string>
   </property>
  </action>
  <action name="actionManual_Center_of_Rotation">
   <property name="text">
    <string>Manual Center of Rotation</string>
   </property>
  </action>
  <action name="actionFilters">
   <property name="text">
    <string>Filters</string>
   </property>
  </action>
  <action name="actionOnlineDocumentation">
   <property name="text">
    <string>Online Documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionTomopyRecon">
   <property name="text">
    <string>TomoPy Reconstruction</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About MantidImaging</string>
   </property>
  </action>
  <action name="actionSavuFilters">
   <property name="text">
    <string>Savu Filters</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
